collect_app/src/androidTest/java/org/odk/collect/android/AllWidgetsFormTest.java

```diff
@@ -293,6 +301,7 @@ public void testUrlWidget() {
 
         onView(withId(R.id.simple_button)).perform(click());
         onView(withText("URL widget")).perform(swipeLeft());
+        //check!!?
```

Comment:
If this comment needs to be here, it should have more context for someone else to understand!

Reply:
I'm sorry @lognaturel, i completely forgot to remove that comment. That was just for me. I'll do that now.